tests/npm-app/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "private": true,
   "scripts": {
-    "build": "del-cli -f dist/ && rollup --config && cp app.json dist/",
-    "demo": "npm run build && http-server -o demo/index.html"
+    "build": "del-cli -f dist/ && rollup --config && cp app.json dist/"
   },
   "type": "module",
   "dependencies": {
+    "@microsoft/ccf-app": "file:../../js/ccf-app",
     "js-base64": "^3.5.2",
     "jsrsasign": "^10.0.4",
```

Comment:
Do we really need jsrsasign? Feels like it should be easy to re-route into C++ and we would get rid of a dependency with recent CVEs.

Reply:
The dependencies of the npm-app test are really just a test to see whether they work in CCF. I don't think CCF will ever expose all of the functionality of jsrsasign, so including it in the test is still useful as it's a popular library that people may use, and we don't want to accidently do something that breaks it. I think in the long run there should be a more dedicated test suite to test a large number of popular npm packages.